pkgs/tools/system/xmousepasteblock/default.nix

```diff
@@ -0,0 +1,30 @@
+{ xorg, stdenv, fetchgit, libev, pkgconfig }:
+
+stdenv.mkDerivation {
+  name = "XMousePasteBlock-1.0";
+  src = fetchgit {
```

Comment:
Could you use `fetchFromGitHub` instead?

Reply:
Sure